server/src/main/java/org/opensearch/index/codec/PerFieldMappingPostingFormatCodec.java

```diff
@@ -69,6 +69,12 @@ public PerFieldMappingPostingFormatCodec(Mode compressionMode, MapperService map
         this.logger = logger;
     }
 
+    public PerFieldMappingPostingFormatCodec(MapperService mapperService, Logger logger) {
```

Comment:
We probably could drop this constructor and use `Mode.BEST_SPEED` as the default

Reply:
This change was added